code/mob/new_player.dm

```diff
@@ -232,6 +232,21 @@ mob/new_player
 				boutput(usr, "<span class='notice'>There is an administrative lock on entering the game!</span>")
 				return
 
+			if(living_pop_cap)
+				var/living_people = 0
+				for(var/client/C in clients)
+					LAGCHECK(LAG_LOW)
```

Comment:
sleeping seems quite dangerous, considering the clients list is constantly having elements added and removed as players connect and disconnect

Reply:
Fair enough there, removed the lag check.